scripts/abi_artifact_generator/templates/functions.inc.jinja_template

```diff
@@ -57,6 +57,11 @@ NOINLINE /*( return_value_type )*/ ABIDEF test_/*( name )*/(void) {
   _Static_assert(sizeof(/*( return_value_type )*/) <= /*( maximum_argument_size )*/,
                  "Size of the return value exceeds the allowed maximum.");
   /*( return_value_type )*/ returned = *(/*( return_value_type )*/ *) expected_state;
+
+#ifdef _MSC_VER
```

Comment:
Why only for MSVC?
Comment.

Reply:
Turns out, different compilers like different thing.

From what I gathered, for MSVC it's better to dump return value address _after_ it's been returned, and _before_ for GCC, hence this pair of `#ifdef _MSC_VER` here and `#ifndef _MSC_VER` in `setup.inc`.

Reply:
(done, BTW)